test/e2e/app-dir/parallel-routes-layouts/parallel-routes-layouts.test.ts

```diff
@@ -0,0 +1,84 @@
+import { createNextDescribe } from 'e2e-utils'
```

Comment:
does this PR need changes from turbopack?

new tests should in theory run by default and it seems like everything is passing, so it might be fine?

Reply:
It looks like this is behaving properly in Turbopack 👍